src/library_sockfs.js

```diff
@@ -207,12 +201,6 @@ addToLibrary({
               opts = subProtocols;
             }
 
-            // some webservers (azure) does not support subprotocol header
-            if (runtimeConfig && null === Module['websocket']['subprotocol']) {
-              subProtocols = 'null';
-              opts = undefined;
-            }
```

Comment:
Separate change? (I don't seem to see this code moved anywhere else.)

Reply:
I specially handle the `=== null` case above.